crates/gpui/src/platform/keystroke.rs

```diff
@@ -318,10 +318,18 @@ fn is_printable_key(key: &str) -> bool {
 impl std::fmt::Display for Keystroke {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         if self.modifiers.control {
-            f.write_char('^')?;
+            if cfg!(target_os = "macos") {
+                f.write_char('^')?;
+            } else {
+                write!(f, "ctrl-")?;
+            }
         }
         if self.modifiers.alt {
-            f.write_char('⌥')?;
+            if cfg!(target_os = "macos") {
+                f.write_char('⌥')?;
+            } else {
+                write!(f, "alt-")?;
+            }
         }
         if self.modifiers.platform {
             #[cfg(target_os = "macos")]
```

Comment:
looks like I forgot to change `platform` to be `super` in Linux instead of this `❖` symbol I've never seen before.